war/src/main/less/modules/form.less

```diff
@@ -310,7 +310,7 @@
   border-radius: 100%;
 
   &::before {
-    content: "?";
+    content: "";
```

Comment:
hmm this might be intentional so that screen readers announce it, cc @janfaracik

Reply:
Hmm had a look at it and it seems all good to be removed 👍 Not too sure why I set the contents twice, good spot @NotMyFault. 

Would be good in the future to replace the link text with a Jenkins Symbol (eg ? in SVG form) so that the appearance of the link is consistent across browsers/fonts, and then adding some invisible text with `.jenkins-visually-hidden` to help screenreaders - although no need for that to be done in this PR.

Cheers :)